src/modules/twap/services/__snapshots__/createTwapOrderTxs.test.ts.snap

```diff
@@ -1,5 +1,64 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Create TWAP order When sell token is NOT approved AND token needs zero approval, then should generate 2 approvals and creation transactions 1`] = `
+Array [
+  "createWithContext",
+  Array [
+    Object {
+      "handler": "0x910d00a310f7Dc5B29FE73458F47f519be547D3d",
+      "salt": "0x00000000000000000000000000000000000000000000000000000015c90b9b2a",
+      "staticInput": "0x00000000000000000000000091056d4a53e1faa1a84306d4deaec71085394bc8000000000000000000000000b4fbf271143f4fbf7b91a5ded31805e42b2208d6000000000000000000000000b4fbf271143f4fbf7b91a5ded31805e42b2208d600000000000000000000000000000000000000000000000000000007c2d24d55000000000000000000000000000000000000000000000000000000000001046a00000000000000000000000000000000000000000000000000000000646b782c0000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000025800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+    },
+    "0x0899c7DC280363d263Cc954506134F249CceC4a5",
+    "0x",
+    true,
+  ],
+]
+`;
+
+exports[`Create TWAP order When sell token is NOT approved AND token needs zero approval, then should generate 2 approvals and creation transactions 2`] = `
+Array [
+  "approve",
+  Array [
+    "0xB4FBF271143F4FBf7B91A5ded31805e42b222222",
+    "100000000000",
+  ],
+]
+`;
+
+exports[`Create TWAP order When sell token is NOT approved AND token needs zero approval, then should generate 2 approvals and creation transactions 3`] = `
+Array [
+  "approve",
+  Array [
+    "0xB4FBF271143F4FBf7B91A5ded31805e42b222222",
+    "0",
+  ],
+]
+`;
+
+exports[`Create TWAP order When sell token is NOT approved AND token needs zero approval, then should generate 2 approvals and creation transactions 4`] = `
+Array [
+  Object {
+    "data": "0xAPPROVE_TX_DATA",
+    "operation": 0,
+    "to": "0x91056D4A53E1faa1A84306D4deAEc71085394bC8",
+    "value": "0",
+  },
+  Object {
+    "data": "0xAPPROVE_TX_DATA",
+    "operation": 0,
+    "to": "0x91056D4A53E1faa1A84306D4deAEc71085394bC8",
+    "value": "0",
+  },
+  Object {
+    "data": "0xCREATE_COW_TX_DATA",
+    "operation": 0,
+    "to": "0xF487887DA5a4b4e3eC114FDAd97dc0F785d72738",
+    "value": "0",
+  },
+]
+`;
```

Comment:
not a fan of snapshots, but not going to complain here given the deadlines we have